modules/migration-mtc-cr-manifests.adoc

```diff
@@ -0,0 +1,363 @@
+  name: <directimagemigration_name>
+spec:
+  srcMigClusterRef:
+    name: <source_cluster_ref> <1>
+    namespace: openshift-migration
+  destMigClusterRef:
+    name: <destination_cluster_ref> <2>
+    namespace: openshift-migration
+  namespaces:
+  - <namespace> <3>
+----
+<1> Specify the `MigCluster` CR name of the source cluster.
+<2> Specify the `MigCluster` CR name of the destination cluster.
+<3> Specify one or more namespaces containing images to be migrated.
+
+[id="directimagestreammigration_{context}"]
+== `DirectImageStreamMigration`
+
+`DirectImageStreamMigration` copies image stream references directly from the source cluster to the destination cluster.
+
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: DirectImageStreamMigration
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: <directimagestreammigration_name>
+spec:
+  srcMigClusterRef:
+    name: <source_cluster_ref> <1>
+    namespace: openshift-migration
+  destMigClusterRef:
+    name: <destination_cluster_ref> <2>
+    namespace: openshift-migration
+  imageStreamRef:
+    name: <image_stream_name> <3>
+    namespace: <source_image_stream_namespace> <4>
+  destNamespace: <destination_image_stream_namespace> <5>
+----
+<1> Specify the `MigCluster` CR name of the source cluster.
+<2> Specify the `MigCluster` CR name of the destination cluster.
+<3> Specify the image stream name.
+<4> Specify the image stream namespace on the source cluster.
+<5> Specify the image stream namespace on the destination cluster.
+
+[id="directvolumemigration_{context}"]
+== `DirectVolumeMigration`
+
+`DirectVolumeMigration` copies persistent volumes (PVs) directly from the source cluster to the destination cluster.
+
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: DirectVolumeMigration
+metadata:
+  name: <directvolumemigration_name>
+  namespace: openshift-migration
+spec:
+  createDestinationNamespaces: false <1>
+  deleteProgressReportingCRs: false <2>
+  destMigClusterRef:
+    name: host <3>
+    namespace: openshift-migration
+  persistentVolumeClaims:
+  - name: <pvc_name> <4>
+    namespace: <pvc_namespace> <5>
+  srcMigClusterRef:
+    name: <source_cluster_ref> <6>
+    namespace: openshift-migration
+----
+<1> `createDestinationNamespaces`: Creates namespaces for the PVs on the destination cluster if `true`.
+<2> `deleteProgressReportingCRs`: Deletes `DirectVolumeMigrationProgress` CRs if `true`. The default value is `false` so that `DirectVolumeMigrationProgress` CRs are retained for troubleshooting.
+<3> `host`: Update the destination cluster name if necessary.
+<4> Specify one or more PVCs to be migrated with direct volume migration.
+<5> Specify the namespace of each PVC.
+<6> Specify the `MigCluster` CR name of the source cluster.
+
+[id="directvolumemigrationprogress_{context}"]
+== `DirectVolumeMigrationProgress`
+
+`DirectVolumeMigrationProgress` shows the progress of the `DirectVolumeMigration` CR.
```

Comment:
Is there a `status` field for this object? Usually that's where status related k8s fields live.

Other than that, it isn't clear how this might show the progress of the migration.

Reply:
No, The DVM CR shows migration status at a high level. The DVM progress CR shows the rsync pods. That part is critical.